src/utils/i18nCh.ts

```diff
@@ -64,7 +66,7 @@ export default class i18nCh {
       }
 
       const projectRootNamespaces = this.loadNamespacesFromPath(path.join(this.localesFolder, "en"));
-      const nodeModulesNamespaces = this.loadNamespacesFromPath(path.join(NODE_MODULES_LOCALES_PATH, "en"));
+      const nodeModulesNamespaces = this.loadNamespacesFromPath(path.join(this.nodeModulesFolder, "en"));
 
       return [...new Set([...projectRootNamespaces, ...nodeModulesNamespaces])];
```

Comment:
This would make the spread and concatenation unnecessary:
```
private loadAllNamespaces(): string[] {
   const allNamespaces = [
      this.loadNamespacesFromFolder(this.localesFolder),
      this.loadNamespacesFromFolder(this.nodeModulesFolder)
   ];

   return [...new Set(allNamespaces.flat())];
}
```
Note that 
- the above is missing the:
```
      if (!this.localesFolder) {
         return [];
      }
```
which was introduced with the previous [PR](https://github.com/companieshouse/ch-node-utils/pull/14/files) because `this.localesFolder` at that point is [already validated](https://github.com/companieshouse/ch-node-utils/blob/efb6cdac4280eab88420fbd81453f9dc68ae2023/src/utils/i18nCh.ts#L18-L22)
- the replicated "path.join" and "en" are here supposed to stay as common factor in the "loadNamespacesFromPath" which could then be:
```
private loadNamespacesFromFolder(baseFolder: string): string[] {
   const folderPath = path.join(baseFolder, "en");
   try {
         return fs.readdirSync(folderPath)
         .filter(file => path.extname(file) === ".json")
         .map(file => path.basename(file, ".json"));
      } catch (error) {
         log(`Error reading directory ${folderPath}: ${error}`);
         return [];
      }
   }
}
```

Reply:
Good point, less duplicate code and removes the unnecessary early return, have addressed in  https://github.com/companieshouse/ch-node-utils/pull/15/commits/635819530c81ae48886f75f02db5fb92a6bf423b